### ISSUE TITLE
Project names matched in completion popup are not entered into text field

### ISSUE THREAD

**Issue by aperezdc** [Igalia/phpreport#319]
**Steps to reproduce**

1. Create a new task (or edit an existing one), and focus the “Project” field.
1. Type part of the name of the project.
1. Wait for the popup list to come up with the matching projects, make sure at least one project is shown in the popup.
1. Press `Tab` to move to the next field.

**Expected outcome**

The first matched project name from the popup is entered in the “Project” input text field. Then the focus moves to the next input.

**Actual outcome**

The focus moves to the next input _without_ placing the matched project name from the popup list into the text field.

**Additional information**

I believe this started happening after #302 was merged.

**Comment by jaragunde** [comment]
> I believe this started happening after #302 was merged.

It started with #288, at that point the typeAhead behavior of that combo box was removed, because it didn't fit well when you search for some customer name in the project box (which is allowed since that PR).

I will write a new typeAhead behavior for that specific combo box.

**Comment by jaragunde** [comment]
The following patch tunes the project combo in several ways, I expect it to fix the issue:
```
commit bd91643f22c082d4d4646543a0d817fade22f248
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Wed Oct 19 17:58:24 2016 +0200

    [#319] Fill project combo box while typing or when tabbing away
    
    Implemented two behaviors in this patch:
    
    * TypeAhead: like other combo boxes, the value is set to the first
      entry found as you type but the search text is not removed
    
    * Select highlighted entry on tab, it combines with typeAhead by
      removing the search text and setting the project name when tab is
      pressed
